src/eventsRelay.js

```diff
@@ -13,10 +13,13 @@ const sendTx = require('./txHelpers/sendTx');
 const { handleEvents } = require('./utils');
 
 module.exports = async (txServerPort, bridgeState) => {
+  const delay = 2000;
```

Comment:
why 2 seconds?

Reply:
500 ms wasn't enough to make the integration tests pass. later we should have a reorg-resistent delay as described here: https://github.com/leapdao/leap-node/issues/140